src/Config/Validator/Deploy/DatabaseSplitConnection.php

```diff
@@ -42,9 +43,14 @@ public function __construct(
 
     /**
      * @return Validator\ResultInterface
+     * @throws ConfigException
      */
     public function validate(): Validator\ResultInterface
     {
+        $splitTypes = $this->stageConfig->get(DeployInterface::VAR_SPLIT_DB);
+        if (empty($splitTypes)) {
```

Comment:
even if SPLIT_DB is empty we should notify customer that we won't use custom configurations for split.

Reply:
@NadiyaS 
I disagree
Since if the client has already split the databases using the custom configuration with remote databases without 'SPLIT_DB' property, that we do not ignore those connections configuration.

Reply:
What if he does not have split DB? 
And in AC to this task was said that we do not support custom connection. Yes, we not remove already existed split, but we do not add new. And as a result we ignore custom connection for split.

Reply:
Ok, Done